openfl-workspace/torch_llm_horovod/src/InHorovodrun.py

```diff
@@ -50,7 +50,7 @@ def get_args():
 
 def main():
     logger = getLogger(__name__)
-    fx.setup_logging(level="INFO", log_file=None)
+    setup_logging
```

Comment:
Do the 2 functions have the exact same functionality?
Also is the `()` not required here?

Reply:
Thanks for pointing this out. I missed adding (). and yes the functionality is same for both function apart from importing tensorflow in native api which is not required here.